cql3/expr/expression.cc

```diff
@@ -593,15 +594,15 @@ value_list get_IN_values(const ::shared_ptr<term>& t, size_t k, const query_opti
     if (auto dv = dynamic_pointer_cast<lists::delayed_value>(t)) {
```

Comment:
Same comment about `using std::views::transform` two lines above

Reply:
I'll remove it.